addons/api/addon/models/target.js

```diff
@@ -53,7 +53,11 @@ export default class TargetModel extends GeneratedTargetModel {
    * instances).  Unresolvable instances are excluded from the array.
    * @type {[CredentialLibraryModel, CredentialModel]}
    */
-  @computed('brokered_credential_source_ids.[]', 'store')
+  @computed(
+    'brokered_credential_source_ids',
+    'brokered_credential_source_ids.[]',
```

Comment:
It seems like if we were to use ember arrays it's would be automatically tracked according to the [docs](https://guides.emberjs.com/release/upgrading/current-edition/tracked-properties/#toc_arrays)